setup.sh

```diff
@@ -5,20 +5,29 @@ then
 fi
 
 MODULE_NAME=$1
-if ! [[ "$MODULE_NAME" =~ ^.*\/conduit-connector-(.*)$ ]]
+if ! [[ "$MODULE_NAME" =~ ^github.com\/.*\/conduit-connector-(.*)$ ]]
 then
-  echo "Module name '$MODULE_NAME' not in required format 'example/conduit-connector-connectorname'"
-  exit 1
+  echo "Module name ${MODULE_NAME} not in recommended format github.com/repository/conduit-connector-connectorname.\n"
+  echo "Certain things (such as pull request templates) will not work correctly."
+  while true; do
+      read -p "Are you sure you want to continue? [y/n]" yn
```

Comment:
We can just read 1 char and continue. And I added a space between the text and the input.
```suggestion
      read -n1 -p "Are you sure you want to continue? [y/n] " yn
```